src/main/java/mate/academy/springboot/datajpa/model/Category.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.springboot.datajpa.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Data

```

Comment:
It won't be too bad in your case, but it's not recommended to use data annotation with entities.
Here you can read more about it:
https://habr.com/ru/company/haulmont/blog/564682/